### PR TITLE
Added cobertura support

### PR DESCRIPTION
**Pull request by zachkknowbe4**
Added remote cobertura support so that https://github.com/Codeception/Codeception/pull/5994 can be merged

### CODE REVIEW

File: c3.php [hunk 1]
```diff
@@ -75,6 +75,7 @@ class_alias('SebastianBergmann\CodeCoverage\Report\Text', 'PHP_CodeCoverage_Repo
     class_alias('SebastianBergmann\CodeCoverage\Report\PHP', 'PHP_CodeCoverage_Report_PHP');
     class_alias('SebastianBergmann\CodeCoverage\Report\Clover', 'PHP_CodeCoverage_Report_Clover');
     class_alias('SebastianBergmann\CodeCoverage\Report\Crap4j', 'PHP_CodeCoverage_Report_Crap4j');
+	class_alias('SebastianBergmann\CodeCoverage\Report\Cobertura', 'PHP_CodeCoverage_Report_Cobertura');
```
Comment (Naktibalda) [inline c3.php:78]: This alias is not necessary, because `PHP_CodeCoverage_Report_Cobertura` class has never existed.

File: c3.php [hunk 2]
```diff
@@ -170,6 +171,14 @@ function __c3_build_crap4j_report(PHP_CodeCoverage $codeCoverage, $path)
         return $path . '.crap4j.xml';
     }
 
+	function __c3_build_cobertura_report(PHP_CodeCoverage $codeCoverage, $path)
+	{
+		$writer = new PHP_CodeCoverage_Report_Cobertura();
```
Comment (Naktibalda) [inline c3.php:176]: 1. Please check if SebastianBergmann\CodeCoverage\Report\Cobertura exists before using it.

2. Your code was indented using tabs, please use 4 spaces instead.

File: c3.php [hunk 3]
```diff
@@ -170,6 +171,14 @@ function __c3_build_crap4j_report(PHP_CodeCoverage $codeCoverage, $path)
         return $path . '.crap4j.xml';
     }
 
+	function __c3_build_cobertura_report(PHP_CodeCoverage $codeCoverage, $path)
```
Comment (Naktibalda) [inline c3.php:174]: This function is defined, but not used,
add `cobertura` route to switch statement: https://github.com/Codeception/c3/blob/8f432510c9b0e7293c6ec1fe5d56e261cba1a018/c3.php#L315-L351

**Review by Naktibalda — CHANGES_REQUESTED** [review]

File: c3.php (hunk 1 above)
Comment (zachkknowbe4) [reply c3.php]: Okay, I've removed the alias

File: c3.php (hunk 2 above)
Comment (zachkknowbe4) [reply c3.php]: I've added a check and fixed indentation.

File: c3.php (hunk 3 above)
Comment (zachkknowbe4) [reply c3.php]: I added cobertura to the switch statement.

**Review by zachkknowbe4 — COMMENTED** [review]
Updated the branch with new changes.

File: c3.php [hunk 4]
```diff
@@ -170,6 +170,17 @@ function __c3_build_crap4j_report(PHP_CodeCoverage $codeCoverage, $path)
         return $path . '.crap4j.xml';
     }
 
+    function __c3_build_cobertura_report(PHP_CodeCoverage $codeCoverage, $path)
+    {
+		if (!class_exists(\SebastianBergmann\CodeCoverage\Report\Cobertura::class)) {
+			throw new Exception("Cobertura report requires php-code-coverage >= 9.2");
+		}
+        $writer = new PHP_CodeCoverage_Report_Cobertura();
```
Comment (Naktibalda) [inline c3.php:178]: Wrong class name here,

**Review by Naktibalda — CHANGES_REQUESTED** [review]

**Comment by zachkknowbe4** [comment]
@Naktibalda My bad, I've fixed it to point to the correct report class now.

**Review by Naktibalda — APPROVED** [review]

**Pull request merged**